lib/elixir/test/elixir/task_test.exs

```diff
@@ -493,6 +493,7 @@ defmodule TaskTest do
     test "timeout" do
       assert catch_exit([:infinity] |> Task.async_stream(&sleep/1, [timeout: 0]) |> Enum.to_list) ==
              {:timeout, {Task.Supervised, :stream, [0]}}
+      refute_receive _
```

Comment:
We should not use this. This will block the test for 100ms and more on CIs. `refute_received` is fine and if it fails, it will reveal a bug (from time to time).